package/gnss_convertors/gnss_convertors.mk

```diff
@@ -4,11 +4,11 @@
 #
 ################################################################################
 
-GNSS_CONVERTORS_VERSION = v0.3.56
+GNSS_CONVERTORS_VERSION = b0fc489e8956224906049e9ea7a678e38fb5c55d
```

Comment:
I assume this is the tip of gnss_converters? if so, can we tag another release and point at a specific release tag? Same for librtcm below.

Reply:
This actually still points to the dev branch. After they are merged to master, will naturally tag them as releases.